src/Dapplo.Microsoft.Extensions.Hosting.AppServices/ResourceNamedPipeServer.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Dapplo and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO.Pipes;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+#if NET472
```

Comment:
The project only supports dotnet core 3.1, is there a reason for this?

Reply:
Oh, I just noticed, it's a left-over from me 🤣

Reply:
Yeah... i should have removed that.